tktermwidget/tkterm.py

```diff
@@ -104,21 +103,22 @@ def __init__(self, master: Misc, autohide: bool = True, *args, **kwargs):
         self.index = 1
         self.current_process: Popen | None = None
 
-        # Bind events
+        # Bind events & tags
         self.text.bind("<Up>", self.up, add=True)
         self.text.bind("<Down>", self.down, add=True)
         self.text.bind("<Left>", self.left, add=True)
```

Comment:
The left and BackSpace could be put into a small loop to remove some redundancy but this may cause more hassle than it fixes. I think either way this could be sorted a little

Reply:
So what should I do?

Reply:
Maybe another improvement but I think we should fix it in another pull request.